frontend/src/main.js

```diff
@@ -26,6 +26,8 @@ import './assets/css/bc-gov-style.css'
 import { library } from '@fortawesome/fontawesome-svg-core'
 import {
   faSort,
+  faFilter,
```

Comment:
Do these imports need to be in-lined? As in are they too long for the linter? They should probably be alphabetized as well